priam/src/main/java/com/netflix/priam/defaultimpl/StandardTuner.java

```diff
@@ -188,6 +194,46 @@ private void addExtraCassParams(Map<String, Object> map) {
         }
     }
 
+    private void writeCassandraSnitchProperties() {
+        String rackdcPropFileName = cassandraConfiguration.getCassHome() + "/conf/" + SnitchProperties.RACKDC_PROPERTY_FILENAME;
+        File rackdcPropFile = new File(rackdcPropFileName);
+        Properties properties = new Properties();
+
+        // Read the existing properties, if any.
+        if (rackdcPropFile.exists()) {
+            try (Reader reader = new FileReader(rackdcPropFile)) {
+                properties.load(reader);
+            } catch (Exception e) {
+                throw new RuntimeException("Unable to read " + SnitchProperties.RACKDC_PROPERTY_FILENAME, e);
+            }
+        }
+
+        // Set the "dc_suffix" property if there is one configured
+        String dcSuffix = cassandraConfiguration.getDataCenterSuffix();
+        if (Strings.isNullOrEmpty(dcSuffix)) {
+            properties.remove("dc_suffix");
+        } else {
+            properties.put("dc_suffix", "_" + dcSuffix);
```

Comment:
well, I'm biased to keep it with the ops as we have flexibility to have either _ or - or anything. If we make sure we are adding one always when we define the suffix in the ops code, I prefer ops-way.